# Range only localisation methodusing LBL

float32 time		# seconds
float32 range		# meters
float64 x
float64 y
float64 z